lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -128,6 +136,27 @@ def self.download_file_from_tag(repository:, tag:, file_path:, download_folder:)
 
         download_path
       end
+
+      # Creates (or updates an existing) GitHub PR Comment
+      def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier:)
+        comments = github_client.issue_comments(project_slug, pr_number)
+
+        existing_comment = comments.detect do |comment|
```

Comment:
Completely opinionated but I personally prefer `find` over `detect` (they are aliases of each other)
```suggestion
        existing_comment = comments.find do |comment|
```